packages/sentry/README.md

```diff
@@ -0,0 +1,148 @@
+# @monkvision/sentry
+This package provides an abstraction layer for the sentry features in the MonkJs ecosystem. If you plan on using any
+of these features, you can use this package to properly set up the sentry inside your application.
```

Comment:
Replace this with :

This package exports a monitoring adapter that can be used to configure the monitoring of MonkJs projects using the `@monkvision/monitoring` project. The adapter exported by this packages uses Sentry, a monitoring platform used by MonkJs to log, report errors and measure performances in its project.